config/dispatcher/dispatcher.ex

```diff
@@ -134,15 +134,15 @@ defmodule Dispatcher do
   end
 
   get "/public-services/*path" do
-    forward conn, path, "http://resource/public-services/"
+    forward conn, path, "http://cache/public-services/"
```

Comment:
**Issue:** In the docs an [ADR](https://github.com/lblod/app-lpdc-digitaal-loket/blob/master/docs/adr/0005-do-not-cache-instantie-overview.md) is documented on why the cache service is not used here. I assume this ADR is still relevant and think that we should **not** change this (and related paths) to the cache service, at least not without further analysis.

Reply:
Aha okay, I did not know about the ADRs. I will revert the routes for `public-services` and `concept-display-configurations` back to `mu-cl-resources` directly.